### PR TITLE
warpd: init at 1.0.2-beta

### PR DESCRIPTION
**Pull request by Enzime**
###### Description of changes

https://github.com/rvaiya/warpd

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/accessibility/warpd/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  nativeBuildInputs = [
+    xorg.libX11 xorg.libXtst xorg.libXi xorg.libXext.dev xorg.libXft
+  ];
+
+  installPhase = ''
+    install -Dm755 bin/warpd $out/bin/warpd
+  '';
+
+  meta = {
+    description = "A modal keyboard-driven virtual pointer for X11.";
```
Comment (iblech) [inline pkgs/applications/accessibility/warpd/default.nix:23]: I think our convention is to go without the trailing end-of-sentence period.

**Comment by iblech** [comment]
Thank you for your work! I just tested this, compilation works fine. I'm not familiar with warpd and <Alt>+<Shift>+<x> didn't work for me, but I also have an unusual setup (no desktop manager, xmonad as window manager). From the GitHub repository, I got the impression that there is a manpage. Is that right? If so, it would be nice if we could install it as well. There is `installManPage` for that.

Otherwise, LGTM, would merge this (after the two minor fixes) if I had the permissions. :-)

**Comment by poita66** [comment]
@Enzime Thanks for packaging warpd.

I noticed that warpd can be compiled for X11 or Wayland. Should this package maybe be renamed to warpd-x11 just so there's no confusion?

**Comment by poita66** [comment]
@rvaiya Just bringing you in on this, as you mentioned that you wanted to be contacted if anyone was interested in packaging warpd for another distro

**Comment by Enzime** [comment]
Obsoleted by #181300

**Pull request closed without merging**